openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -251,3 +259,12 @@ func (e *extension) cleanupOldResources(ctx context.Context, ns string) error {
 	}
 	return nil
 }
+
+// SRVKS-1264 - cleanup of old internal TLS secrets
+func (e *extension) cleanupOldSecrets(ctx context.Context, ns string) error {
```

Comment:
Maybe move this to `cleanupOldResources` ? If it gets too long we split that method later.

Reply:
Updated per suggestion, thanks!